runtime/derive/Cargo.toml

```diff
@@ -7,5 +7,5 @@ edition = "2018"
 proc-macro = true
 
 [dependencies]
-syn = { version = "0.15", features = ["full"] }
-quote = "0.6"
+syn = { version = "1.0", features = ["full"] }
```

Comment:
I just updated this because I was trying some things on our derive crate and I couldn't do it because of the version.

I may use this updates later. Even if I don't, I think its good to keep up to date.

Reply:
If tests are happy, I'm happy.